int32 motion_end
